### PR TITLE
Generate sassdoc for storybook

### PR DESCRIPTION
**Pull request by akomiqaia**
## Describe your changes

move sassdocs to storybook for old components.

It's a lot of files. but all of them are `sassdoc.md` or `sassdoc.stories.mdx` files. A few components needed to have brand-specific folders since their sass documentation is brand-specific. The components that were brand-specific are: 
- 'o-quote'
- 'o-loading'
- 'o-table'
- 'o-footer-services'
- 'o-header-services'
- 'o-typography'

### CODE REVIEW

File: components/g-audio/stories/sassdoc.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import {Meta, Markdown} from "@storybook/blocks";
+import SASSDOC from './docs/sassdoc.md';
+
+<Meta
+title="components/g-audio/sassDoc"
```
Comment (notlee) [inline components/g-audio/stories/sassdoc.stories.mdx:5]: Can we capitalise again please `SassDoc`? 😅 
http://sassdoc.com/

File: components/o-audio/stories/docs/sassdoc.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# o-audio Sass Documentation
+## Mixins
+### oAudio
+Output All oMessage Features
```
Comment (notlee) [inline components/o-audio/stories/docs/sassdoc.md:4]: Hah! I'm not checking all the SassDoc but looks like this is wrong. Could you update the actual comment in Sass too? 🙇 

```suggestion
Output All oAudio Features
```

**Comment by notlee** [comment]
The registry gives an overview in the sidebar which I think for SassDoc is really helpful for finding what you need quickly. Especially as a reference, where you might know what you want but have forgotten the details. I wonder if there's some MarkDown tool we could use to generate a table of contents with links at the top of the SassDoc?

Screenshot 2024-02-14 at 09 37 09

File: components/o-audio/stories/docs/sassdoc.md (hunk 2 above)
Comment (akomiqaia) [reply components/o-audio/stories/docs/sassdoc.md:4]: yeah this is a mistake in our sass mixin doc

**Comment by frshwtr** [comment]
It appears there's some broken tags on the JSDoc pages. 

image

**Comment by akomiqaia** [comment]
> It appears there's some broken tags on the JSDoc pages.
> 
> image

hmmm... this is fine on chrome but not on firefox

**Comment by notlee** [comment]
That's interesting @akomiqaia. It must be that the generated html is invalid, and Firefox is therefore misinterpreting it. Let's focus on getting SassDon out and come back to that?

**Comment by akomiqaia** [comment]
> That's interesting @akomiqaia. It must be that the generated html is invalid, and Firefox is therefore misinterpreting it. Let's focus on getting SassDoc out and come back to that?

Sassdocs are updated now and if everyone is happy. ready to go. Added TOC(table of content for each sassdoc to recreate the sidebar)

**Review by notlee — APPROVED** [review]
That's great :D

**Pull request merged**